backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/radiobutton/BpkRadioButton.kt

```diff
@@ -83,11 +84,42 @@ fun BpkRadioButton(
     modifier = rowModifier,
     verticalAlignment = Alignment.CenterVertically,
   ) {
-    BpkRadioButton(selected = selected, onClick = onClick, enabled = enabled, interactionSource = interactionSource)
-    BpkText(
-      text = text,
-      style = BpkTheme.typography.footnote,
-      color = if (enabled) BpkTheme.colors.textPrimary else dynamicColorOf(BpkColor.SkyGrayTint04, BpkColor.BlackTint06),
+    BpkRadioButtonImpl(selected = selected, onClick = onClick, enabled = enabled, interactionSource = interactionSource)
+    val contentColor =
+      if (enabled) BpkTheme.colors.textPrimary else dynamicColorOf(BpkColor.SkyGrayTint04, BpkColor.BlackTint06)
+    CompositionLocalProvider(
+      LocalContentColor provides contentColor,
+      LocalContentAlpha provides 1f,
```

Comment:
I'm thinking maybe we shouldn't include setting content alpha here. The widgets which override it usually override it for its content and have good reasons for it.